### PR TITLE
importorder

### PR DESCRIPTION
**Pull request by enqueue**

### CODE REVIEW

**Comment by wiebeck** [comment]
What format is this file? Something for IntelliJ IDEA or Eclipse? How can I "apply" this file and what issue does this fix?

**Comment by enqueue** [comment]
> What format is this file?

The format of this file is key / value text file. The keys are unique numbers signifying the place of the value pattern in the import order of Java class files.

> Something for IntelliJ IDEA or Eclipse?

You can use it in Eclipse. I do not know if you can import it in IntelliJ IDEA.

> How can I "apply" this file

Either in your IDE or in the spotless gradle task.

> and what issue does this fix?

Frustration with misconfigured IDEs.

File: java/riege.importorder [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#Organize Import Order
+#Thu Jul 20 16:55:53 CEST 2023
+11=\#java
+12=\#javax
+13=\#junit
```
Comment (wiebeck) [inline java/riege.importorder:5]: Isn't jakarta missing at this position?

File: java/riege.importorder [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#Organize Import Order
+#Thu Jul 20 16:55:53 CEST 2023
+11=\#java
```
Comment (wiebeck) [inline java/riege.importorder:3]: Shouldn't `io` be the first one here as well?

File: java/riege.importorder (hunk 1 above)
Comment (enqueue) [reply java/riege.importorder:5]: I will double-check with the Checkstyle configuration.

File: java/riege.importorder (hunk 2 above)
Comment (enqueue) [reply java/riege.importorder:3]: I will double-check with the Checkstyle configuration.

**Review by wiebeck — APPROVED** [review]

**Pull request merged**